tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/MultiFunctionApplicationHelpers.csproj

```diff
@@ -180,21 +211,30 @@
     <PackageReference Include="Microsoft.Extensions.Logging.Configuration" Version="6.0.0" Condition="'$(TargetFramework)' == 'net6.0'" />
     <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="6.0.0" Condition="'$(TargetFramework)' == 'net6.0'" />
     <PackageReference Include="Serilog.Extensions.Hosting" Version="4.2.0" Condition="'$(TargetFramework)' == 'net6.0'" />
+
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="7.0.0-preview.6.22324.4" Condition="'$(TargetFramework)' == 'net7.0'" />
```

Comment:
Hmm... not sure about the preview 6 reference here

Reply:
I'm not sure what that's about either. I'll replace it.